x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -997,7 +997,7 @@ describe('when on the endpoint list page', () => {
         const subHeaderBackLink = await renderResult.findByTestId('flyoutSubHeaderBackButton');
         expect(subHeaderBackLink.textContent).toBe('Endpoint Details');
         expect(subHeaderBackLink.getAttribute('href')).toEqual(
-          `${APP_PATH}${MANAGEMENT_PATH}/endpoints?page_index=0&page_size=10&selected_endpoint=1`
+          `${APP_PATH}${MANAGEMENT_PATH}/endpoints?page_index=0&page_size=10&selected_endpoint=1&show=details`
```

Comment:
😄 I see why you added the `show` to that one routing method now. If I remember correctly, I think i had remove it a few commits ago because of the reason I mentioned above - its the default, and we probably should prune defaults params from the URL

Reply:
Okay, then we do want to keep that pattern where default URL params are pruned. Even though keeping it here is consistent? @paul-tavares

Reply:
it's something we can talk about perhaps at one of our eng. office hours. The pruning of URL params was first done during implementation of Trusted Apps and I kind of like it (I like clean urls 😁 ), but lets discuss with the broader team.